### ISSUE TITLE
Jaeger dependencies page does not work as expected on jaeger-production-template.yml

### ISSUE THREAD

**Issue by jkandasa** [jaegertracing/jaeger-openshift#19]
Jaeger `Dependencies` page does not work as expected on jaeger-production-template.yml
Always throws the following exception,
![image](https://user-images.githubusercontent.com/1004403/27137407-84bf1496-513b-11e7-869c-14667bbf4f84.png)
```
Request URL:https://jaeger-query-jaeger.xyz.xip.io/api/dependencies?endTs=1497450385942&lookback=604800000
```
##### Response:
```json
{"data":null,"total":0,"limit":0,"offset":0,"errors":[{"code":500,"msg":"Error reading dependencies from storage: Cannot execute this query as it might involve data filtering and thus may have unpredictable performance. If you want to execute this query despite the performance unpredictability, use ALLOW FILTERING"}]}
```

**Comment by yurishkuro** [comment]
The CQL error is odd and should be looked at (sounds like a schema mismatch), but even if it worked there wouldn't be any data returned since we don't have the job yet that aggregates the data for dependencies diagram.

**Comment by pavolloffay** [comment]
I have a reproducer it's because of C* version used in `jpkroehling/cassandra`

```bash
ccm create test -v 3.0.12 -n 3 -s
MODE=test sh plugin/storage/cassandra/schema/create.sh | ccm node1 cqlsh
go run cmd/query/main.go  --query.static-files=jaeger-ui-build/build/ -cassandra.keyspace=jaeger_v1_test
```

after running create.sh you will see ``<stdin>:179:ConfigurationException: Unable to find custom indexer class 'org.apache.cassandra.index.sasi.SASIIndex'.

I think `jaeger-cassandra-schema-job` should fail if there is such an error.

@jpkrohling could you please look at it? There are two parts clarify or use standard C* image (related to #11 ) and fix schema job to fail in there is an error.

**Comment by pavolloffay** [comment]
also related issue is https://github.com/uber/jaeger/issues/244

**Comment by pavolloffay** [comment]
Fixed in #24
